Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -929,6 +959,19 @@ def delete_user(user_key):
     return 'User {} have been deleted.'.format(command_args['userKey'])
 
 
+def list_labels(user_key):
+    command_args = {
+        'userKey': user_key,
+    }
+    service = get_service(
+        'gmail',
+        'v1',
+        ['https://www.googleapis.com/auth/gmail.readonly'])
+    results = service.users().labels().list(userId=user_key).execute()
+    labels = results.get('labels', [])
```

Comment:
```suggestion
    labels = results.get('labels') or []
```

Reply:
Is there a reason for this? The get method allows you to specify a default if nothing is returned.

Reply:
This is incase where ```results.get('labels')``` can return None (if the api returns something like 
```
{
   "labels": null
}
``` 

In that case even if you defined a default value of a list, the labels variable would still get None. 
Many apis behave like that, I am not sure regarding the gmail api though. This is just to be on the safe side here.
In case the api doesn't behave like this you can just keep the way it was before

Reply:
This is what the function handles, i.e. in your example if the labels == None then it would still assign [] to the labels var.

Reply:
```results.get('labels', [])``` handles the case where the labels key does appear in the response/dict.

Reply:
Ah yes, well spotted. Gmail has at least 1 label as you can't delete the default labels `INBOX` etc...

Reply:
Alright then, if so you can keep the current query.